sunpy/instr/goes.py

```diff
@@ -1268,3 +1271,100 @@ def _calc_xraylum(flux, date=None):
     else:
         xraylum = 4 * np.pi * sun.constants.au.to("m")**2 * flux
     return xraylum
+
+def flareclass_to_flux(flareclass):
```

Comment:
This functionality is already in sunpy: 
```python
>>> from sunpy.util import goes_flare_class
>>> print(goes_flare_class('X23.1'))
0.00231
```
This needs to delete the other or merge them.